e2e/tests/ui-driven/src/create-flow/create-flow.spec.ts

```diff
@@ -175,6 +182,33 @@ test.describe("Navigation", () => {
       "some data field",
     );
 
+    nextNode = page.locator(".hanger > a").nth(12);
```

Comment:
This looks like a good use case for a helper function and `last()` ([docs](https://playwright.dev/docs/api/class-locator#locator-last)).

If it's not as simple as using `last()`, we could also maybe chain a `filter()` to narrow it down a little.

```ts
const getNextNode = () => page.locator(".hanger > a").last()
let nextNode = getNextNode();
```

Reply:
Great suggestion, that works well! 👍